scripts/kind-dev-env.sh

```diff
@@ -23,16 +23,10 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 : "${IMAGE_REGISTRY:=ghcr.io/llm-d}"
 
 # Set a default VLLM_SIMULATOR_IMAGE if not provided
-: "${VLLM_SIMULATOR_IMAGE:=llm-d-inference-sim}"
-
-# Set a default VLLM_SIMULATOR_TAG if not provided
-export VLLM_SIMULATOR_TAG="${VLLM_SIMULATOR_TAG:-latest}"
+export VLLM_SIMULATOR_IMAGE="${VLLM_SIMULATOR_IMAGE:-ghcr.io/llm-d/llm-d-inference-sim:latest}"
 
 # Set a default EPP_IMAGE if not provided
-: "${EPP_IMAGE:=llm-d-inference-scheduler}"
-
-# Set a default EPP_TAG if not provided
-export EPP_TAG="${EPP_TAG:-dev}"
+export EPP_IMAGE="${EPP_IMAGE:-ghcr.io/llm-d/llm-d-inference-scheduler:dev}"
```

Comment:
It is very useful to keep the TAG separate from the image name. Please undo this change